### ISSUE TITLE
insert every 10 minutes

### ISSUE THREAD

**Issue by cooliobr** [futzu/x9k3#16]
"How do I insert every 10 or 15 minutes?"

**Comment by cooliobr** [comment]
Adrian, sorry it is my fault.
thanks


Att.

Rafael Machado
*HALLO! Midia Group.*




Em qui., 15 de jun. de 2023 às 20:06, Adrian ***@***.***>
escreveu:

**Comment by futzu** [comment]
Sorry, I didn't mean to be such a dick, I was having a bad day. 
That being said, that is the way you do it. There no method from inserting at fixed intervals, 
If you think about it, shows never have the commercials at fixed intervals.

**Comment by cooliobr** [comment]
Don't worry, I know what it's like to have a bad day. I also appreciate
your response.
I recently returned from a long period in the hospital, and it's been
difficult for me to readjust.
However, I've already found a solution by using MySQL with the event
scheduler, and it worked really well.


Att.

Rafael Machado
*HALLO! Midia Group.*




Em seg., 19 de jun. de 2023 às 23:24, Adrian ***@***.***>
escreveu:
